docs/cloud-get-started.md

```diff
@@ -0,0 +1,52 @@
+---
+id: cloud-get-started
+title: Getting Started
+---
+
+## Register on the ReSolve Cloud Platform
+
+Go to the [admin.resolve.sh](https://admin.resolve.sh) page, click **Sign Up** and follow through the registration process. After registration, you will gain access to your personal admin panel.
+
+## Install the ReSolve CLI
+
+Type the following command to install the reSolve cloud CLI to your application.
+
+```bash
+yarn global add resolve-cloud
+```
+
+After this, use the CLI to authenticate to the reSolve Cloud Platform:
+
+```bash
+resolve-cloud login
+```
+
+The CLI will ask you for you account credential and establish an authentication session.
+
+## Create a ReSolve Application
+
+Create a reSolve application. For this tutorial, you can use one of examples that come with the reSolve framework.
```

Comment:
What do you mean by "for this tutorial"? what tutorial? is the user supposed to create a tutorial?

Reply:
I mean, "To go through this tutorial" (the tutorial that we are editing)